src/controllers/wallet/wallet.controller.js

```diff
@@ -573,7 +573,7 @@ async function mintTokens(req, res) {
   const changeAddress = req.body.change_address || null;
   const mintAuthorityAddress = req.body.mint_authority_address || null;
   const allowExternalMintAuthorityAddress = req.body.allow_external_mint_authority_address || false;
-  const unshiftData = req.body.unshiftData || null;
+  const unshiftData = req.body.unshift_data || null;
```

Comment:
Do we need `|| null` with the default value as `true`? And if yes, should it be `true`? (the same apply to the other)

Reply:
Removed the `|| null`